dsc/tests/dsc_set.tests.ps1

```diff
@@ -67,6 +67,49 @@ Describe 'config set tests' {
         ($result.psobject.properties | Measure-Object).Count | Should -Be 3
     }
 
+    It 'can accept the use of --format <format> as a subcommand' -Skip:(!$IsWindows) -TestCases @(
+        @{ format = 'yaml'; expected = @'
+beforeState:
```

Comment:
For tests, we should stop using the registry resource since it only works on Windows.  You should use the [`Test/Echo`](https://github.com/PowerShell/DSC/blob/main/tools/dsctest/dscecho.dsc.resource.json) resource instead.

Reply:
had trouble using Test/Echo with set and "text: hello" as input so ended up using the resource from the set tests that works on all platforms